### PR TITLE
[ISSUE-608] Skip block volume tests in short CI suite

### LINKED ISSUE #608: Remove Block tests in long CI
Community tests for Block Mode needs 18+ minutes for execution. They hasn't been a part of short CI

### PR DESCRIPTION
**Pull request by safronovD**
### Resolves #608 

- Add flag for `est/e2e/baremetal_e2e_test.go` to execute all tests
- Skip block volume and other fs types tests for short CI suite

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
_Provide test details_

### CODE REVIEW

File: test/e2e/scenarios/controller-node-fail.go [hunk 1]
```diff
@@ -47,6 +47,10 @@ func DefineControllerNodeFailTestSuite(driver testsuites.TestDriver) {
 }
 
 func controllerNodeFailTest(driver testsuites.TestDriver) {
+	ginkgo.BeforeEach(func() {
```
Comment (rkiselenko) [inline test/e2e/scenarios/controller-node-fail.go:50]: Just pass a closure
```golang
ginkgo.BeforeEach(skipIfNotAllTests)
```

**Review by rkiselenko — CHANGES_REQUESTED** [review]

File: test/e2e/scenarios/controller-node-fail.go (hunk 1 above)
Comment (safronovD) [reply test/e2e/scenarios/controller-node-fail.go]: fixed, thanks!

**Review by rkiselenko — APPROVED** [review]

File: test/e2e/scenarios/bare-metal-testdriver.go [hunk 2]
```diff
@@ -83,8 +83,18 @@ func (d *baremetalDriver) GetDriverInfo() *testsuites.DriverInfo {
 
 // SkipUnsupportedTest is implementation of TestDriver interface method
 func (d *baremetalDriver) SkipUnsupportedTest(pattern testpatterns.TestPattern) {
-	if pattern.VolType == testpatterns.InlineVolume || pattern.VolType == testpatterns.PreprovisionedPV {
-		e2eskipper.Skipf("Baremetal Driver does not support InlineVolume and PreprovisionedPV -- skipping")
+	if !d.needAllTests {
+		if pattern.VolMode == corev1.PersistentVolumeBlock {
```
Comment (mishoyama) [inline test/e2e/scenarios/bare-metal-testdriver.go:87]: Please add comments here why we skip this tests.

Comment (safronovD) [reply test/e2e/scenarios/bare-metal-testdriver.go:88]: comments added

**Review by mishoyama — APPROVED** [review]

**Pull request merged**